shell/platform/darwin/macos/framework/Source/FlutterView.mm

```diff
@@ -113,7 +115,6 @@ - (BOOL)layer:(CALayer*)layer
 }
 
 - (void)shutdown {
```

Comment:
This is empty now, should it be removed?

Reply:
Removed.